clients/web/src/stylesheets/body/plugins.styl

```diff
@@ -1,5 +1,7 @@
 @import 'nib'
 
+$warnColor = #f0ad4e
+
```

Comment:
I agree that we should have only one warning color, I actually think `.g-plugin-warning` is a misnomer, though... it's actually more of an error message. Would you be OK if I added a `.g-plugin-error` class to the existing element and moved these styles to it?